resources/views/home.blade.php

```diff
@@ -1,50 +1,182 @@
+                    </filter>
+                    </defs>
+                </svg>
+            </div>
         </div>
     </div>
-</div>
+</section>
+
 
-<div class="container-fluid bg-white">
-    <div class="row justify-content-center">
-        <div class="col-md-12 my-5">
-            <h1 class="text-center">
-                @lang('home.text_subscribe_cta', ['link' => 'sms://+1' . env('TWILIO_FROM_NUMBER') . ';?&body=' . __('home.text_subscribe_keyword')])
-            </h1>
-            <h2 class="text-center">
-                @lang('home.add_to_contacts_cta', ['link' => 'tel:+1' . env('TWILIO_FROM_NUMBER')])
-            </h2>
+<section id="steps" class="max-w-5xl mx-auto px-8 py-10 sm:py-20">
+    <div class="sm:flex -mx-4">
+        <div class="sm:w-1/3 px-4 mb-8 sm:mb-0">
+            <h2 class="text-2xl font-medium font-display">Step 1</h2>
+            <p>@lang('home.step_1')</p>
         </div>
-    </div>
-    <div class="row justify-content-center">
-        <div class="col-md-12 my-5">
-            <h1 class="text-center">
-                <strong>@lang('home.locale_support_head')</strong>
-            </h1>
-            <h4 class="text-center">
-                @lang('home.locale_support_body', ['link' => 'tel:+1' . env('TWILIO_FROM_NUMBER')])
-            </h4>
+        <div class="sm:w-1/3 px-4 mb-8 sm:mb-0">
+            <h2 class="text-2xl font-medium font-display">Step 2</h2>
+            <p>@lang('home.step_2')</p>
+        </div>
+        <div class="sm:w-1/3 px-4">
+            <h2 class="text-2xl font-medium font-display">Step 3</h2>
+            <p>@lang('home.step_3')</p>
         </div>
     </div>
-</div>
+</section>
 
-<div class="container-fluid">
-    <div class="row justify-content-center">
-        <div class="col-md-12 my-5">
-            <h2 class="text-center">@lang('home.partners_head')</h2>
-            <p class="text-center">
-                @lang('home.partners_body')
-            </h2>
-        </div>
+<section class="px-8 py-16 sm:py-32 bg-red-200 relative">
+    <div class="absolute inset-0" style="background-image: url({{ asset('img/map.jpg') }})"></div>
+    <div class="bg-white max-w-md mx-auto px-10 py-5 text-center rounded-lg shadow-lg relative z-10">
+        <h2 class="text-3xl sm:text-4xl font-medium text-center font-display leading-tight mb-4">
+            @lang('home.locale_support_head')
+        </h2>
+        <p>@lang('home.locale_support_body', ['link' => 'tel:+1' . env('TWILIO_FROM_NUMBER')])</p>
     </div>
-</div>
+</section>
 
+<section class="max-w-5xl mx-auto py-10 sm:py-20 px-8">
+    <div class="px-4">
+        <div class="sm:flex items-center -mx-4">
+            <div class="px-4 sm:w-1/2 mb-8 sm:mb-0">
+                <img class="mx-auto mb-6" style="max-width: 190px" src="{{ asset('img/kmuw.png') }}" alt="KMUW logo">
+                <img class="mx-auto" style="max-width: 190px" src="{{ asset('img/devict.png') }}" alt="devICT logo">
```

Comment:
I tried clicking on these when I was playing around with it. Any reason not to make them links?

Also, we should include [AB&C Bilingual Services](https://www.abcbilingualresources.com/) as a partner here too. They (Claudia) are doing the translations!

Reply:
Added AB&C and all of them are links now